index.html

```diff
@@ -366,6 +366,26 @@ <h5><span class="fa fa-warning"></span>
                 </div>
               </div>
 
+              <div class="panel panel-default"> <!-- Gentoo -->
+                <div class="panel-heading">
+                  <h4 class="panel-title">
+                    <a data-toggle="collapse" data-parent="#distros" href="#gentoo">
+                      <img class="distro-logo" src="img/distros/gentoo.png" alt="Gentoo Logo"/>&nbsp; Gentoo
+                    </a>
+                  </h4>
+                </div>
+                <div id="gentoo" class="panel-collapse collapse">
+                  <div class="panel-body">
+                    <p>To install:</p>
+<code><span class="fa fa-dollar"></span> emerge -av --autounmask sys-apps/openrazer</code></pre>
+                    <p></p>
+                    <p>If You will not add the desired drivers to "/etc/modules-load.d",
+                    then You will have to "modprobe" them manually.</p>
```

Comment:
`razer_mount` script (triggered by udev rule) should load the module if necessary, are you sure this needs to be done manually?